README.md

```diff
@@ -270,6 +272,8 @@ function MyUserInterface({name, tagline, imageUrl, homepage, size}) {
 }
 ```
 
+> Try this out in your browser [here](https://codesandbox.io/s/EJ4kmOV4)!
```

Comment:
I think the glamorous logo looks kinda funny in the circle there. Why don't you find a funny cat picture or something to use instead ;-) And the tagline can be anything else :)